openedx/adg/lms/webinars/views.py

```diff
@@ -16,8 +16,11 @@
 
 
 def webinar_description_page_view(request, pk):
-    # TODO This is a temp view and will be properly functional in LP-2623
-    return render(request, 'adg/lms/webinar/description_page.html', context={'pk': pk})
+    """
+    A view to get description about a specific webinar.
+    """
+    return render(request, 'adg/lms/webinar/description_page.html',
+                  context={'webinar': get_object_or_404(Webinar, pk=pk)})
```

Comment:
Can we use generic CBV for example [DetailView](https://docs.djangoproject.com/en/2.2/ref/class-based-views/generic-display/#detailview), we can update URL accordingly as well

Reply:
+1